components/google_drive/sources/changes-to-specific-files/changes-to-specific-files.js

```diff
@@ -29,31 +31,35 @@ module.exports = {
       label: "Files",
       description: "The files you want to watch for changes.",
       optional: true,
-      async options({ prevContext }) {
+      default: [],
+      options({ prevContext }) {
         const { nextPageToken } = prevContext;
-        if (this.drive === "myDrive") {
-          return await this.googleDrive.listFiles({ pageToken: nextPageToken });
-        }
-
-        return await this.googleDrive.listFiles({
-          pageToken: nextPageToken,
-          corpora: "drive",
-          driveId: this.drive,
-          includeItemsFromAllDrives: true,
-          supportsAllDrives: true,
-        });
+        const baseOpts = {};
+        const opts = this.drive === "myDrive"
```

Comment:
Does it make sense to have a method like `isMyDrive` to capture this logic? Or perhaps a `MY_DRIVE_LABEL` constant to capture the "myDrive" string?